SQL_assignment/sql_assignment1

```diff
@@ -0,0 +1,82 @@
+CREATE TABLE: Creates an Employees table with columns for EmployeeID, FirstName, LastName, Department, and Salary.
+
+    Create table emp(EmpoyeeID int primary key,firstname varchar(20),lastname varchar(20),department varchar(20),salery int)
+
+
+
+INSERT: Adds three employee records to the table.
+
+    insert into emp values(1,'akash','j','IT',50000)
+    insert into emp values(,2'arun','p','NON-IT',32000)
+    insert into emp values(31,'Abhishek','s','IT'38000)
+
+
+SELECT: Shows different ways to query data:
+
+    select* from emp
+    select Department,FirstName from emp where EmployeeID=1
+    select Firstname ,salary from emp
+
+
+Select all columns and rows
+  
+    select* from emp
+
+
+Select specific columns
+
+    select Firstname ,salary from emp
+
+
+Select with a WHERE clause to filter results
+
+    select Department,FirstName from emp where EmployeeID=1
+
+
+What is the purpose of the IDENTITY keyword in the CREATE TABLE statement?
+
+    IDENTITY keyword is used to auto increment columns
+    IDENTITY(1,1) means it will start with 1 and when each data is enterd its value will be increased by 1
+
+
+Write a SELECT statement to retrieve only the FirstName and Salary of all employees.
+
+    select FirstName,Salary from emp
```

Comment:
You created the table with column "salery", and wrote a query for retrieving column "salary."  Will this work?

Reply:
sorry that was a spelling error i corrected in the code but forgot to update in the answer file